app/root.tsx

```diff
@@ -185,15 +184,32 @@ export default function App() {
               <Atlas />{" "}
               <Overlay>
                 <Show above="lg">
-                  <FilterMenu>
+                  <FilterMenu defaultIndex={0}>
                     <AdminDropdowns />
                   </FilterMenu>
                 </Show>
-                <Outlet
-                  context={{
-                    children: AdminDropdowns(),
-                  }}
-                />
+                <Hide above="lg">
+                  <FilterMenu>
+                    <AdminDropdowns />
+                  </FilterMenu>
+                </Hide>
+                <Flex
+                  direction={{ base: "column-reverse", lg: "column" }}
+                  justify={{ base: "flex-start", lg: "space-between" }}
+                  align={"flex-end"}
```

Comment:
Adding `pointerEvents={"none"}` here should fix the issue where this Flex is blocking interaction with the map.